pkg/routeimporter/config.go

```diff
@@ -6,16 +6,44 @@ import (
 )
 
 type ConfigRouteEntry struct {
-	Url       string               `json:"url"`
-	Layout    *fragment.Definition `json:"layout"`
-	Fragments fragment.Collection  `json:"fragments"`
-	Metadata  map[string]string    `json:"metadata"`
+	Url               string               `json:"url"`
+	LayoutTemplate    *fragment.Definition `json:"layout"`
+	FragmentTemplates fragment.Collection  `json:"fragments"`
+	Metadata          map[string]string    `json:"metadata"`
+	IgnoreValidation  bool                 `json:"ignoreValidation"`
 }
 
 func LoadRoutes(server *viewproxy.Server, routeEntries []ConfigRouteEntry) error {
 	for _, routeEntry := range routeEntries {
-		server.Get(routeEntry.Url, routeEntry.Layout, routeEntry.Fragments, viewproxy.WithRouteMetadata(routeEntry.Metadata))
+		layout := createFragment(routeEntry.LayoutTemplate)
+
+		fragments := make(fragment.Collection, len(routeEntry.FragmentTemplates))
+		for i, fragmentTemplate := range routeEntry.FragmentTemplates {
+			fragments[i] = createFragment(fragmentTemplate)
+		}
+
+		err := server.Get(
+			routeEntry.Url,
+			layout,
+			fragments,
+			viewproxy.WithRouteMetadata(routeEntry.Metadata),
+		)
+
+		if err != nil {
+			return err
+		}
 	}
 
 	return nil
 }
+
+func createFragment(template *fragment.Definition) *fragment.Definition {
+	fragment := fragment.Define(template.Path, fragment.WithMetadata(template.Metadata))
+	fragment.TimingLabel = template.TimingLabel
+
+	if template.IgnoreValidation {
+		fragment.IgnoreValidation = true
+	}
```

Comment:
Would seem okay to pass this through the same as `TimingLabel`

```suggestion
	fragment.IgnoreValidation = template.IgnoreValidation
```